### PR TITLE
i18n(pt-br): Update `api-reference`

### PR DESCRIPTION
**Pull request by Fryuni**
#### Description

Update the Brazilian Portuguese translation of the Runtime API reference.

#### Related issues & labels

- Unblocks #5553

### CODE REVIEW

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 1]
```diff
@@ -757,27 +757,77 @@ const { page } = Astro.props;
 - `/postagens/[page].astro` geraria as URLs `/postagens/1`, `/postagens/2`, `/postagens/3`, etc.
 - `/postagens/[...page].astro` geraria as URLs `/postagens`, `/postagens/2`, `/postagens/3`, etc.
 
+`paginate()` recebe os seguintes arguments:
```
Comment (Fryuni) [inline src/content/docs/pt-br/reference/api-reference.mdx:760]:
```suggestion
`paginate()` recebe os seguintes argumentos:
```

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 2]
```diff
@@ -757,27 +757,77 @@ const { page } = Astro.props;
...
+- `pageSize` - O número de itens exibido por página
```
Comment (Fryuni) [inline src/content/docs/pt-br/reference/api-reference.mdx:761]:
```suggestion
- `pageSize` - O número de itens exibidos por página
```

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 3]
```diff
@@ -1028,20 +1078,109 @@ const { Content, headings, remarkPluginFrontmatter } = await entry.render();
...
+
+Uma união de strings de todos os nomes de coleções `type: 'content'` definidos em seu arquivo `src/content/config.*`.
+
+#### `DataCollectionKey`
+
+Uma união de strings de todos os nomes de coleções `type: 'data'` definidoes em seu arquivo `src/content/config.*`.
+
+#### `SchemaContext`
+
+O objeto `context` que `defineCollection` para a função `shape` do `schema`. Esse tipo pode ser útil ao criar schemas reutilizáveis para múltiplas coleções.
```
Comment (Fryuni) [inline src/content/docs/pt-br/reference/api-reference.mdx:1107]: Não é o método `shape` 🤦

```suggestion
O objeto `context` que `defineCollection` passa para a construção do `schema`. Esse tipo pode ser útil ao criar schemas reutilizáveis para múltiplas coleções.
```

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 4]
```diff
@@ -1100,6 +1252,133 @@ Este componente providencia syntax highlighting de linguagens específicas para
...
+
+Formato a ser usado como um valor de fallback para a tag `<img>`. Por padrão, é configurado como `.png` para imagens estáticas, `.gif` para imagens animadas e `.svg` para arquivos SVG.
+
+##### `pictureAttributes`
+
+Use essa propriedade para aplicar atributos ao elemento externo `<picture>`. Atributos aplicados diretamente ao componente `<Picture />` serão aplicados internamente ao elemento `<img>`, exceto aqueles usados para a transformação da imagem.
+
+### `<Content />`
+
+Um componente genérico utilizado para renderizar o conteúdo de uma [entrada de uma coleção de conteúdo](pt-br/guides/content-collections/#o-que-são-coleções-de-conteúdo).
```
Comment (sarah11918) [inline src/content/docs/pt-br/reference/api-reference.mdx:1347]:
```suggestion
Um componente genérico utilizado para renderizar o conteúdo de uma [entrada de uma coleção de conteúdo](/pt-br/guides/content-collections/#o-que-são-coleções-de-conteúdo).
```

File: src/content/docs/pt-br/reference/api-reference.mdx
[the remark this answers is not in the document]
Comment (Fryuni) [reply src/content/docs/pt-br/reference/api-reference.mdx:1]: Done!

File: src/content/docs/pt-br/reference/api-reference.mdx [hunk 5]
```diff
@@ -452,6 +451,28 @@ const pedidos = Array.from(Astro.locals.pedidos.entries());
...
+
+Essa propriedade só está disponível ao fazer o build em modo SSR (renderização do lado do servidor) e não deve ser utilizada em sites estáticos.
+
+### `Astro.preferedLocaleList`
+
+`Astro.preferedLocaleList` representa a lista de todos o slocales que são tanto requisitados pelo navegador quanto suportados pelo seu website. Isso produz uma lista de todas as linguagens compatíveis entre o seu site e seu visitante.
+
+Se nenhuma das linguagens requesitadas pelo navegador forem encontradas na sua lista de idiomas, então o valor é `[]`: você não suporta nenhum dos idiomas de preferência do seu visitante.
+
+Se o navegador não especificar nenhuma linguagem de preferência, então esse valor será igual ao [`i18n.locales`](/pt-br/reference/configuration-reference/#i18nlocales): todas os seus idiomas suportados serão considerados igualmente aceitáveis pelo visitante sem nenhuma preferênca.
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/reference/api-reference.mdx:468]:
```suggestion
{// TODO: Add configuration-reference/#i18nlocales link back. }
Se o navegador não especificar nenhuma linguagem de preferência, então esse valor será igual ao `i18n.locales`: todas os seus idiomas suportados serão considerados igualmente aceitáveis pelo visitante sem nenhuma preferênca.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @Fryuni 🙌

**Pull request merged**